envergo/templates/moulinette/result.html

```diff
@@ -35,13 +35,47 @@ <h1 class="fr-mb-5w">Simulation réglementaire du projet</h1>
             aria-controls="share-modal"
             data-btn="bottom">Partager cette page par email</button>
   </div>
-
+  <div class="print-only">
+    <h2 class="fr-h3">Caractéristiques du projet</h2>
+
+    {% include '_map_snippet.html' with map=moulinette.map map_id="summary" %}
+
+    <ul class="fr-mb-3w">
+      {% if moulinette.raw_data.created_surface %}
+        <li>
+          <strong>Surface nouvellement impactée par le projet :</strong> {{ moulinette.raw_data.created_surface }} m²
+          <br />
+          <span class="fr-hint-text">Bâti, voirie, espaces verts, remblais et bassins — temporaires et définitifs</span>
+        </li>
+      {% endif %}
+      {% if moulinette.raw_data.existing_surface %}
+        <li>
+          <strong>Surface déjà impactée avant le projet :</strong> {{ moulinette.raw_data.existing_surface }} m²
+          <br />
+          <span class="fr-hint-text">Bâti, voirie, espaces verts, remblais et bassins</span>
+        </li>
+      {% endif %}
+
+      {% if moulinette.raw_data.final_surface %}
+        <li>
+          <strong>Surface impactée totale, y compris l'existant :</strong> {{ moulinette.raw_data.final_surface }} m²
+          <br />
+          <span class="fr-hint-text">Bâti, voirie, espaces verts, remblais et bassins — temporaires et définitifs</span>
+        </li>
+      {% endif %}
+      {% if moulinette.raw_data.is_lotissement %}
+        <li>
+          <strong>Le projet concerne-t-il un lotissement ?</strong> {{ moulinette.raw_data.is_lotissement }}
+        </li>
+      {% endif %}
```

Comment:
This summary is more simple than the one from AR, but it seems enough to me. 

I take the data from `moulinette.raw_data`, but maybe it is better from `moulinette.catalog` ?